submissions/oluwaboriShobayo/DOM/productPage.js

```diff
@@ -0,0 +1,79 @@
+let  Sform = document.getElementById("form");
+     quantity = document.getElementById("quantity")
+     shippingCost = document.getElementById("shippingCost");
+     totalCost = document.getElementById("totalCost");
+     button = document.getElementById("button");
+  let price = document.getElementById("price")
+ 
+
+    function numberValidation (event) {
+    if (quantity.value == '') {
+        button.disabled = true;
+        button.style.backgroundColor = "#808080"
+        quantity.nextElementSibling.innerHTML = "Please enter a valid quantity";
+       event.preventDefault()
+        return false;
+    }   
+    let numRegex = /^[1-9]/;
+    if (!quantity.value.match(numRegex)) {
+        button.disabled = true;
+        button.style.backgroundColor = "#808080"
+        quantity.nextElementSibling.innerHTML = "Please enter a valid quantity";
+        event.preventDefault();
+        return false;
+    } 
+    else {
+        quantity.nextElementSibling.innerHTML = "";
+        button.disabled = false;
+        return true;
+        
+    }
+}
+
+function calculateproductPrice () {
+    let priceperUnit = 8000;
+    if (quantity.value >=1) {
+        let productPrice = (priceperUnit * quantity.value)
+        price.innerHTML = productPrice
+        calculateShippingCost(productPrice)
+        calculateTotalCost(productPrice)
+    }else if (quantity.value == '') {
```

Comment:
Is this check relevant? Because you are already saying on line 10, it should throw an error message. I will advise you just put this in an `else` statement instead of `else if`